man/fi_verbs.7.md

```diff
@@ -170,6 +184,14 @@ The verbs provider checks for the following environment variables.
 : The prefix or the full name of the network interface associated with the verbs
   device (default: ib)
 
+*FI_VERBS_MR_CACHE_ENABLE*
+: Enable Memory Registration caching (default: 0)
+
+*FI_VERBS_MR_CACHE_SIZE*
+: Maximum number of cache entries (default: 4096)
+
+*FI_VERBS_MR_CACHE_LAZY_SIZE*
+: Minimum memory region size for lazy deregistration (default: 0)
```

Comment:
Please verify if this is a minimum size for caching or a maximum size.  That is, are regions larger than this size cached or not cached.

Reply:
The terminology here is somewhat confusing, with "CACHE_SIZE" means the # of entries while "CACHE_LAZY_SIZE" means a threshold of buffer size above which the registration is cached. One of both of them may need to be renamed to present a clearer interface.

Reply:
@j-xiong `FI_VERBS_MR_CACHE_SIZE` -> `FI_VERBS_MR_CACHE_ENT_NUM` or something like that?

Reply:
@gladkovdmitry17 That's in the right direction I think. We could completely avoid using SIZE here. For example: `FI_VERBS_MR_CACHE_SIZE` -> `FI_VERBS_MR_CACHE_ENTRIES`, `FI_VERBS_MR_CACHE_LAZY_SIZE` -> `FI_VERBS_MR_CACHE_THRESHOLD`.